databroker/_drivers/mongo_embedded.py

```diff
@@ -189,12 +188,29 @@ def __init__(self, datastore_db, *, handler_registry=None, root_map=None,
         datastore_db : pymongo.database.Database or string
             Must be a Database or a URI string that includes a database name.
         handler_registry : dict, optional
-            Maps each asset spec to a handler class or a string specifying the
-            module name and class name, as in (for example)
-            ``{'SOME_SPEC': 'module.submodule.class_name'}``. If None, the
-            result of ``databroker.core.discover_handlers()`` is used.
-        root_map : dict, optional
-            Maps resource root paths to different paths.
+            This is passed to the Filler or whatever class is given in the
+            filler_class parametr below.
+            Maps each 'spec' (a string identifying a given type or external
+            resource) to a handler class.
+            A 'handler class' may be any callable with the signature::
+                handler_class(resource_path, root, **resource_kwargs)
+            It is expected to return an object, a 'handler instance', which is also
+            callable and has the following signature::
+            handler_instance(**datum_kwargs)
+            As the names 'handler class' and 'handler instance' suggest, this is
+            typically implemented using a class that implements ``__init__`` and
+            ``__call__``, with the respective signatures. But in general it may be
+            any callable-that-returns-a-callable.
+        root_map: dict, optional
+            This is passed to Filler or whatever class is given in the filler_class
+            parameter below.
+             str -> str mapping to account for temporarily moved/copied/remounted
+            files.  Any resources which have a ``root`` in ``root_map`` will be
+            loaded using the mapped ``root``.
+        filler_class: type, optional
+            This is Filler by default. It can be a Filler subclass,
+            ``functools.partial(Filler, ...)``, or any class that provides the
+            same methods as ``DocumentRouter``.
```

Comment:
The `handler_registry`, `root_map` and `filler_class` are not explicitly declared as arguments. Should they be defined?

Reply:
I would have personally preferred to leave them as explicitly declared args, but I think @gwbischof did this on @tacaswell's suggestion and I don't feel strongly.

Reply:
To be clear, they don't technically have to be declared because they are captured by the base class, `Broker`. The reason one might declare them anyway is just to make the signature more obvious / self-describing.

Reply:
It would be cleaner to either (i) declare them explicitly via keyword arguments and keep the docstrings or (ii) assume they are passed via `**kwargs` to the base class, and remove the corresponding docstrings. Right now we ave a mixture of (i) and (ii). I am in favor or either of them, not the mixture.

Reply:
I went with solution (i)